test_settings.py

```diff
@@ -32,7 +32,7 @@ def root(*args):
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'edx_rbac',
-    'tests',
```

Comment:
Why did you rename this app?

Reply:
It wasn't a Django app before, I had to create the Django app with a different name to generate migrations in the same app, as mentioned in the PR description (it works locally without adding migrations) it fails on Django 2.2 in Travis if the app doesn't have migrations with Sqlite.

Reply:
@mraarif where/how are you re-producing this error? I don't see a jenkins build with it and I couldn't reproduce it locally.  I ran into a different discussion related error when I tried to run pytset in edx-platform with this change.

Reply:
https://travis-ci.org/github/edx/edx-rbac/builds/663497804